com.unity.multiplayer.mlapi/Editor/PostProcessScene.cs

```diff
@@ -11,13 +11,7 @@ public class NetworkScenePostProcess : MonoBehaviour
         [PostProcessScene(int.MaxValue)]
         public static void ProcessScene()
         {
-            //If we are in playmode (editor or stand alone) we do not want this to execute
-            if (Application.isPlaying)
-            {
-                return;
-            }
```

Comment:
I think we should separate MLAPI changes from testproject changes.
There are so many things going under this PR and make it much harder to review.